### ISSUE TITLE
Docs for `getRevocationIds` states it is only for debugging

### ISSUE THREAD

**Issue by rickynils** [biscuit-auth/biscuit-haskell#54]
The documentation for [getRevocationIds](https://github.com/biscuit-auth/biscuit-haskell/blob/3ceb3aef6134dee3c12e6eff3354af10868a3b5d/biscuit/src/Auth/Biscuit/Token.hs#L495) states that the function is only supposed to be used for debugging. But there doesn't seem to be any other way to retrieve the revocation id for a token, and according to the [Biscuit docs](https://www.biscuitsec.org/docs/guides/revocation/#how-the-revocation-service-receives-and-stores-data) you are supposed to store the revocation ids for all tokens created. Then I figure `getRevocationIds` is not just for debugging, and the docs are wrong?

**Comment by divarvel** [comment]
You are correct, "debugging" is wrong. The comment was meant with token verification in mind (where `getRevocationId` is indeed not the right way to do it), and debugging here means more or less "getting information about a token for display purposes"

**Comment by divarvel** [comment]
Feel free to PR a better description if you have one in mind, else I'll improve it on Monday

**Comment by divarvel** [comment]
i've changed the wording in #55, let me know if it is better.

(also, don't hesitate to ask if anything else seemed unclear in the docs)
